src/model/UInt64.ts

```diff
@@ -84,6 +84,27 @@ export class UInt64 {
         return uint64.toHex(this.toDTO());
     }
 
+    /**
+     * Get string representation of number (by default, decimal representation)
+     *
+     * @return {string}
+     */
+    public toString(radix: number = 10): string {
+        var result = '';
```

Comment:
can we change to `const` or `let` instead of var?

Reply:
Sure, definitely. Let me edit that commit.